lib/ErrorSerializer.js

```diff
@@ -0,0 +1,20 @@
+var serializeError = require('serialize-error');
```

Comment:
Nah, I don't feel that strongly about it. We can just go with how it is right now.